provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -105,9 +105,9 @@ def execute_df_job(output_uri, code, store_type, spark_configs, credentials, sou
     func_parameters = []
     for location in sources:
         if location.endswith(".csv"):
-            func_parameters.append(spark.read.option("recursiveFileLookup", "true").csv(location))
+            func_parameters.append(spark.read.option("header","true").option("recursiveFileLookup", "true").csv(location))
         elif location.endswith(".parquet"):
-            func_parameters.append(spark.read.option("recursiveFileLookup", "true").parquet(location))
+            func_parameters.append(spark.read.option("header","true").option("recursiveFileLookup", "true").parquet(location))
```

Comment:
could also rewrite to this for possible berevity:
```
    for location in sources:
        reader_builder = spark.read.option("recursiveFileLookup", "true")
        if location.endswith(".csv"):
            reader_builder.csv(location)
        elif location.endswith(".parquet"):
            reader_builder.parquet(location)
        else:
            raise Exception(f"the file type for '{location}' file is not supported.")
        func_parameters.append(reader_builder)
```

Reply:
but up to you

Reply:
This If statement needs to be reworked for a different bug, will change there